pkg/engine/armvariables.go

```diff
@@ -80,6 +80,62 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 		kubernetesVersion = orchProfile.OrchestratorVersion + AzureStackSuffix
 	}
 
+	provisionScriptParametersCommonString := "[concat(" +
```

Comment:
Thank you @sylr for this code maintenance improvement :)

@CecileRobertMichon @devigned @mboersma  FYI

(Also, I manually validated that nothing was lost during translation.)